zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```
@@ -339,6 +338,14 @@ private void broadcastAll(Message m) {
     }
   }
 
+  private void unicast(Message m, NotebookSocket conn) {
+    try {
+      conn.send(serializeMessage(m));
+    } catch (IOException e) {
+      LOG.error("socket error", e);
```

is it potentially a problem to catch and ignore an error with this?